pkg/cloud/gcp/actuators/util/gcp_credentials.go

```diff
@@ -25,7 +25,6 @@ import (
 	"golang.org/x/oauth2/google"
 	apicorev1 "k8s.io/api/core/v1"
 	apimachineryerrors "k8s.io/apimachinery/pkg/api/errors"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 	controllerclient "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Potentially would have kept `client` over `controllerclient`, but I can see that causing clashes, tough decision 🤔

Reply:
I would agree that having `client` over `controllerclient` is more preferable, as we primary use the first one anywhere else.